.github/actions/setup-binary-builds/action.yml

```
@@ -109,16 +109,28 @@ runs:
           set -euxo pipefail
           CONDA_ENV="${RUNNER_TEMP}/conda_environment_${GITHUB_RUN_ID}"
 
-          conda create \
-            --yes --quiet \
-            --prefix "${CONDA_ENV}" \
-            "python=${PYTHON_VERSION}" \
-            cmake=3.22 \
-            conda-build=3.27.0 \
-            conda=23.10.0 \
-            ninja=1.10 \
-            pkg-config=0.29 \
-            wheel=0.37
+          if [[ "${PYTHON_VERSION}" = "3.12" ]]; then
+            conda create \
+              --yes --quiet \
+              --prefix "${CONDA_ENV}" \
+              "python=3.9" \
```

It should be 3.12 here, shouldn't it?
```suggestion
              "python=3.12" \
```